### PR TITLE
Fix syntax error: `not in ""`. 

### PR DESCRIPTION
**Pull request by TiansuYu**
related with issue: https://github.com/titipata/pubmed_parser/issues/104

There is a syntax error in `medline_parser`. Only needs a one line quick fix, AFAIK.

### CODE REVIEW

**Review by titipata — APPROVED** [review]
LGTM

**Comment by titipata** [comment]
Thanks for the PR!

**Comment by TiansuYu** [comment]
Will you release a new version that one could use?

File: pubmed_parser/medline_parser.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def parse_references(pubmed_article, reference_list):
         return references
     else:
         references = ";".join(
-            [ref["pmid"] for ref in references if ref["pmid"] is not ""]
+            [ref["pmid"] for ref in references if ref["pmid"] != ""]
```
Comment (raypereda-gr) [inline pubmed_parser/medline_parser.py:475]:
```python
 [ref["pmid"] for ref in references if ref["pmid"]]
```
Empty strings are "falsy".  In an `if` statement, you can drop the explicit test for not empty string.
https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty

Comment (TiansuYu) [reply pubmed_parser/medline_parser.py:475]: Thanks for the reminder. I think it also depends on your style: more pythonic or more explicit. Feel free to change this.

Comment (rohangoli) [reply pubmed_parser/medline_parser.py:475]: Thanks for the change!

**Pull request merged**